dist/index.js

```diff
@@ -5488,6 +5505,10 @@ function writeReport(reportPath, report, projectData, contents) {
     return __awaiter(this, void 0, void 0, function* () {
         fs.writeFileSync(path.join(reportPath, "report.md"), contents);
         fs.writeFileSync(path.join(reportPath, "data.json"), JSON.stringify(projectData, null, 2));
+        let latest = path.join(reportPath, "..", "latest");
+        console.log(`creating symbolic link: ${reportPath} ${latest}`);
+        console.log(cp.exec(`unlink "${latest}"`).toString());
```

Comment:
Should this be `execSync` ?
```suggestion
        console.log(cp.execSync(`unlink "${latest}"`).toString());
```

Reply:
intentionally execSync since it's a simple and fast invocation and it's not a server.  tooling expected to be synchronous.  If we did execSync the code bloats much more with callback, building up output, handling events etc... :)